components/ui/Icon.tsx

```diff
@@ -1,52 +1,6 @@
 import { asset } from "$fresh/runtime.ts";
 import type { JSX } from "preact";
-
-export type AvailableIcons =
-  | "ArrowsPointingOut"
-  | "Bars3"
-  | "ChevronLeft"
-  | "ChevronRight"
-  | "ChevronUp"
-  | "ChevronDown"
-  | "CreditCard"
-  | "Deco"
-  | "Diners"
-  | "Discord"
-  | "Discount"
-  | "Elo"
-  | "Facebook"
-  | "FilterList"
-  | "Heart"
-  | "Instagram"
-  | "Linkedin"
-  | "Minus"
-  | "MapPin"
-  | "MagnifyingGlass"
-  | "Mastercard"
-  | "Message"
-  | "Phone"
-  | "Pix"
-  | "Plus"
-  | "QuestionMarkCircle"
-  | "Return"
-  | "Ruler"
-  | "ShoppingCart"
-  | "Star"
-  | "Tiktok"
-  | "Trash"
-  | "Truck"
-  | "Twitter"
-  | "User"
-  | "Visa"
-  | "WhatsApp"
-  | "XMark"
-  | "Zoom"
-  | "Alert"
-  | "AlertInfo"
-  | "AlertSuccess"
-  | "AlertWarning"
-  | "AlertError"
-  | "share";
+import { AvailableIcons } from "./IconTypes.ts";
```

Comment:
I really like to import Icon and type of available from same place. 

That was just a change of your preference?

Reply:
Because my objective would be to change only the types

Reply:
But thinking better, I think I can read the file and do a regex to change just the types

Reply:
Lets keep in separate files, but export at Icons.tsx too? That way solve your and my problem.